tests/test_adders_listed_value_adder.py

```diff
@@ -98,32 +147,23 @@ def test__add_value_with_id_greater_than_current_last_index_plus_one_throws_exce
         )
 
 
-def test__add_to_inventory_of_listed_values_append_to_end_no_custom_values(test_adder):
-    new_value_id = test_adder._add_to_inventory_of_listed_values(
-        feature_id="A-11",
-        new_value_en="New value, listed with a comma",
-        new_value_ru="Есть первые, вторые и третьи",
-        index_to_assign=-1,
-    )
-    assert new_value_id == "A-11-15"
-
-    assert test_adder.output_file_with_listed_values.exists()
-
-    check_existence_of_output_csv_file_and_compare_with_gold_standard(
-        output_file=test_adder.output_file_with_listed_values,
-        gold_standard_file=GS_FILE_WITH_LISTED_VALUES_ADDITION_TO_THE_END_OF_VALUE,
-    )
-
-
-def test__add_to_inventory_of_listed_values_append_to_end_with_explicit_index_no_custom_values(
+def test_add_listed_value_throws_exception_with_index_to_assign_much_greater_than_current_maximum_plus_one(
     test_adder,
 ):
-    # The feature A-11 has 14 values, we are asking the method to add the 15-th one
+    with pytest.raises(ListedValueAdderError, match="must be between 1 and 4"):
+        test_adder.add_listed_value(
+            feature_id="A-1", new_value_en="Value", new_value_ru="значение", index_to_assign=100
+        )
```

Comment:
I think that cases for `0`, `-1` and `100` could be all compressed into one because the error message is the same. But I don't insist.

Reply:
Done!